docker/python3.6_ci/Dockerfile

```diff
@@ -12,6 +12,7 @@ RUN pip install moto==1.0.1
 
 COPY is_up_to_date_with_master.py /app/is_up_to_date_with_master.py
 
+COPY build_lambda.sh /app/build_lambda.sh
```

Comment:
thought: does `COPY . /app` work?

Might save us having to edit the Dockerfile when we add new scripts.